integration/config.yml

```diff
@@ -0,0 +1,151 @@
+# Retains default and configurable name-values for the integration framework
+#
+# All configuration can be set at runtime with environment variables, for example:
+# KEEP_ENVIRONMENTS=OnFail
+# APPS_CHAINLINK_VERSION=1.0.0
+
+network: 'ethereum_geth' # Selected network for test execution
+keep_environments: Never # Options: Always, OnFail, Never
+
+apps:
+  chainlink:
+    image: smartcontract/chainlink
+    version: 0.10.3
```

Comment:
This is very old

Reply:
This value doesn't actually get used, at least on the CI side, but will bump it up once CI is running and this is all merge-able.